### ISSUE TITLE
search modal is too transparent on safari

### ISSUE THREAD

**Issue by signalnerve** [cloudflare/workers-docs-engine#147]
when viewing the search modal on safari, it seems like some styles aren't being applied that are causing it to be extremely transparent and hard to read.

on safari:

<img width="985" alt="Screen Shot 2020-08-17 at 3 35 51 PM" src="https://user-images.githubusercontent.com/922353/90442016-8be9d200-e09f-11ea-9e93-67418bdccf98.png">

on chrome:

<img width="869" alt="Screen Shot 2020-08-17 at 3 35 54 PM" src="https://user-images.githubusercontent.com/922353/90442035-92784980-e09f-11ea-8cce-d1c546db8766.png">

**Comment by adamschwartz** [comment]
After investigating this, it turns out that this is just a straight Safari bug. It’s not crazy surprising as there’s a lot of fancy compositing going on here. But essentially, if you open up safari devtools and uncheck/check the `backdrop-filter` blur application, it fixes the issue. I had a hack that sort of worked by forcing a repaint on every keystroke, but it ended up having the opposite artifacts in a sense: the blur which is only supposed to be behind the dropdown was showing on the backdrop too when an update to the results caused the height of the dropdown to _decrease_.

So bottom line, I decided to temporarily background blur feature from Safari only using (unfortunately) a CSS hack: https://github.com/cloudflare/workers-docs-engine/commit/f132e5802c092aa44405606808b2111dd1b707d9. When I have time I’ll make a minimal repro demo and file a bug on with Safari. Until then... only Chrome will get this nice-to-have effect.

**Comment by adamschwartz** [comment]
Could be https://bugs.webkit.org/show_bug.cgi?id=159573 or similar.
